docs/book/architecture/events.rst

```diff
@@ -0,0 +1,80 @@
+.. index::
+   single: Events
+
+Events
+======
+
+.. tip::
+
+    You can learn more about events in general in `the Symfony documentation <http://symfony.com/doc/current/event_dispatcher.html>`_.
+
+What is the naming convention of Sylius events?
+-----------------------------------------------
+
+The events that are designed for the entities have a general naming convention: ``sylius.entity_name.event_name``.
+
+The examples of such events are: ``sylius.product.pre_update``, ``sylius.shop_user.post_create``, ``sylius.taxon.pre_create``.
+
+
+Events reference
+~~~~~~~~~~~~~~~~
+
+All Sylius bundles are using :doc:`SyliusResourceBundle </bundles/SyliusResourceBundle/index>`, which has some built-in events.
+
++-------------------------------+----------------+
+| Event                         | Description    |
++===============================+================+
+| sylius.<resource>.pre_create  | Before persist |
++-------------------------------+----------------+
+| sylius.<resource>.create      | After persist  |
++-------------------------------+----------------+
+| sylius.<resource>.post_create | After flush    |
++-------------------------------+----------------+
+| sylius.<resource>.pre_update  | Before persist |
```

Comment:
We do not call _persist_ method on manager on update action.. should be ``Before flush`` I suppose.